.github/workflows/prbuild.yml

```diff
@@ -2,6 +2,11 @@ name: Pull Request Build
 
 on:
   pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - ready_for_review
     branches:
       - main
```

Comment:
can we add path filter here in case unnecessary build for docs file?
```
paths:
      - "**"
      - "!/*.md"
      - "!/**.md"
 ```

Reply:
Good point, Ray! Perhaps, .png, .svg, .drawio too. Just in case.

Reply:
Good idea! Thanks

Reply:
@raylrui Hmm, tried that with your config but doesn't seems to work as expected. Tried updating some `.md` but still run the test. Do you have any idea what went wrong?

Reply:
sure, I will have a check again

Reply:
@williamputraintan  Ahh, just found the answer
>when we make the on: pull-request actions, it will automatically use the synchronize event which will always consider all commits of a file change.

So when you make changes out the paths in any commit of this PR, it will always been triggered.
I just have tested on my side, it will works next time when we just have **.md files change in that PR.

https://github.com/actions/runner/issues/2324#issuecomment-1703345084

Reply:
so actually , we can just add path-ignores this time, test next time 🤣

```
paths-ignore:
 - '**.md'
```

Reply:
Ahhh, so the path filtered is from the PR change, not from the commit. That make sense why it ignores the `path-ignore`. Oh well, will just leave it there in case for docs PR. Thank you for looking in to it!